packages/kbn-pm/package.json

```diff
@@ -51,7 +51,6 @@
     "log-symbols": "^2.2.0",
     "mkdirp": "^0.5.1",
     "ora": "^1.4.0",
-    "prettier": "^1.11.1",
```

Comment:
__question:__ hmm, does `node scripts/eslint --fix` from kibana root folder pick up TypeScript files? If I'm not mistaken without `prettier` dependency here, `yarn prettier` from `kbn-pm` folder won't work anymore, or am I missing something?

Reply:
confirmed what @azasypkin said. We my need to update the line?
```json
    "prettier": "prettier --write './src/**/*.ts'"
```

Reply:
this is ugly but works

```json
    "prettier": "../../node_modules/.bin/prettier --write './src/**/*.ts'"
```

Reply:
Another ugly version is below :) but maybe we can just have separate dependency for `kbn-pm` like it was before and remove it once/if we have `tslint` support in Kibana.
```
"prettier": "yarn --cwd ../../ prettier --write './packages/kbn-pm/src/**/*.ts'"
```

Reply:
Haha, damn :( Not sure why it still worked locally for me, but you're totally right. Fixing now.